src/views/movie_list_view.js

```diff
@@ -0,0 +1,98 @@
+import Backbone from 'backbone';
+import Movie from 'models/movie';
+import MovieView from 'views/movie_view';
+import Search from 'models/search';
+
+const MovieListView = Backbone.View.extend({
+  initialize(params){
+    this.template = params.template;
+  },
+
+  events: {
+    'click button.btn-show': 'getRentalMovies',
+    'click button.btn-search': 'getIndividualMovie',
+  },
+
+  getRentalMovies() {
+    event.preventDefault()
+    this.model.reset();
+
+    console.log(" all rental movies");
+
+    //calls to the rails API, returns movies in RailsDB in JSON
+    const movieList = this.model.fetch()
+    movieList.then(() => {
+      movieList.responseJSON.forEach((movie) => {
+        const newMovie = new Movie({
+        title: movie.title,
+        overview: movie.overview,
+        release_date: movie.release_date,
+        image_url: movie.image_url,
+      });
+      // this.model.add(newMovie)
+      //somehow fetch on a collection inside a promise adds the models created
+      //to the collection without having to explicitly add them
+      });
+      this.render();
+      this.$('.btn-add').hide();
+    })
+  },
+
+  getIndividualMovie() {
+    event.preventDefault();
```

Comment:
We will also need to have `event` as a parameter to this function: `getIndividualMovie(event) { ... }`.